errands-routes.go

```diff
@@ -131,18 +128,23 @@ type filteredUpdateReq struct {
 	Delete bool   `json:"delete"`
 }
 
+//nolint:gocognit // TODO
 func (s *ErrandsServer) updateFilteredErrands(c *gin.Context) {
 	key := c.Param("key")
 	value := c.Param("val")
+
 	var updateReq filteredUpdateReq
+
 	if err := c.ShouldBind(&updateReq); err != nil {
 		c.JSON(http.StatusBadRequest, gin.H{
 			"message": "Invalid Parameters",
 			"error":   err.Error(),
 		})
+
 		return
 	}
-	errands, err := s.GetErrandsBy(func(errand *schemas.Errand) bool {
+
+	errands := s.GetErrandsBy(func(errand *schemas.Errand) bool {
```

Comment:
Why are we no longer checking the error for this?

Reply:
Because the error was hard coded to nil on line 243 of the original file.